salvo/src/lib/api/docker_volume.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```

Comment:
(fyi) This file is a good case for us not having a single BUILD target for all these proto files. None of the other proto files seem to be importing this one, so having a separate compilation unit for it makes sense from the code perspective.